views/partials/articles/article.hbs

```diff
@@ -0,0 +1,12 @@
+<h1>Article</h1>
+{{!-- No data has to be passed in because html will be rendered before variables are read in --}}
+<h4>
+  Title: {{articles.item.title}}<br> 
+  Body: {{articles.item.body}}<br>
+  Author: {{articles.item.author}}<br>
+</h4>
+
+<p>
+<form action="" method="get">
```

Comment:
The basic implementation of form submission is to use `<form action="/articles/{{articles.item.title}}/edit">` rather than putting the formaction directly on the input